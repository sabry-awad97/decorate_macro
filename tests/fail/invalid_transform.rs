use decorate_macro::decorate;

fn wrong_params(x: i32) -> i32 {
    x + 1
}

#[decorate(transform_params = wrong_params)]
fn add(x: i32, y: i32) -> i32 {
    x + y
}

fn main() {}
