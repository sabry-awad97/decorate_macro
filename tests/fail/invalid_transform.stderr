error: unexpected end of input, expected identifier
 --> tests/fail/invalid_transform.rs:7:1
  |
7 | #[decorate(transform_params = wrong_params)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `decorate` (in Nightly builds, run with -Z macro-backtrace for more info)
